jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -180,12 +198,28 @@ def search(self, error_on_fail=False):
 
         # QTable stores all the filtered sky coordinate points to be marked
         catalog_results = QTable({'coord': filtered_skycoord_table})
-        self.number_of_results = len(catalog_results)
 
+        self.number_of_results = len(catalog_results)
         # markers are added to the viewer based on the table
         viewer.marker = {'color': 'red', 'alpha': 0.8, 'markersize': 5, 'fill': False}
         viewer.add_markers(table=catalog_results, use_skycoord=True, marker_name=self._marker_name)
 
+        # adding in coordinates and Source IDs into the catalog table
+        if self.catalog_selected == "SDSS":
+            for row in self.app._catalog_source_table:
+                row_info = {'Right Ascension (degrees)': row['ra'],
+                            'Declination (degrees)': row['dec'],
+                            'Object ID': row['objid']}
+                self.table.add_item(row_info)
+
+        if self.catalog_selected == 'From File...':
+            for row in self.app._catalog_source_table:
+                # find new to add in a way to append the source id to the table
+                # 'Object ID': row['label']} ; 'label' is failing tests
```

Comment:
you can always do something like:

```
'Object ID': row.get('label', '')
```

which will then use the "label" column if it exists, and otherwise will fill that entry with an empty string.